cmd/janus/init.go

```diff
@@ -59,8 +59,8 @@ func initStatsdClient(config config.Stats) *stats.StatsdStatsClient {
 		WithField("map", sectionsTestsMap.String()).
 		Debug("Setting stats second level IDs")
 
-	client := stats.NewStatsdStatsClient(config.DSN, config.Prefix)
-	client.SetHttpMetricCallback(stats.NewHasIDAtSecondLevelCallback(sectionsTestsMap))
+	client := stats.NewStatsdClient(config.DSN, config.Prefix)
```

Comment:
`client, err := stats.NewClient(config.DSN, config.Prefix)` - currently `err` is always `nil`, but may change in the future with new backends